core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -246,8 +246,10 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
   }
 
   private def logAuditMessage(principal: KafkaPrincipal, authorized: Boolean, operation: Operation, resource: Resource, host: String) {
-    val permissionType = if (authorized) "Allowed" else "Denied"
-    authorizerLogger.debug(s"Principal = $principal is $permissionType Operation = $operation from host = $host on resource = $resource")
+    if (authorized)
+      authorizerLogger.debug(s"Principal = $principal is Allowed Operation = $operation from host = $host on resource = $resource")
+    else
+      authorizerLogger.warn(s"Principal = $principal is Denied Operation = $operation from host = $host on resource = $resource")
```

Comment:
@ijuma currently default authorizer logger is configured to WARN level in log4j.properties.
And deny logs are sufficient for debugging authorizer related issues. So I want to enable deny logs by default.  I am fine with INFO level, but then I also want INFO level in log4j.properties